rom/dev/doc/test-coverage/test-coverage.md

```diff
@@ -136,6 +136,9 @@ Ensure that boot ROM can load a 128k bundle into ICCM (assert ICCM contents in t
 Test Scenario| Test Name | ROM Error Code
 ---|---|---
 Expand `smoke_test` to perform a hitless update and confirm everything is mixed into the identity correctly. | N/A | N/A
-Stress test: Perform many hitless updates in a row | N/A | N/A
+Validate fix for #817: warm reset during hitless update | N/A | N/A
```

Comment:
Lines 139-142 should be deleted here.

Reply:
I see line 140 is still there? There should be no new additions to rom/dev/doc/test-coverage/test-coverage.md.

Reply:
I wanted to check with you on that, thanks for pointing it out. Removed the line in the latest commit.